firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/core/CrashlyticsController.java

```diff
@@ -780,18 +811,21 @@ boolean finalizeSessions(int maxCustomExceptionEvents) {
    * class.
    */
   private void doOpenSession() throws Exception {
-    final Date startedAt = new Date();
+    final long startedAtSeconds = new Date().getTime() / 1000;
     final String sessionIdentifier = new CLSUUID(idManager).toString();
 
     Logger.getLogger().d(Logger.TAG, "Opening a new session with ID " + sessionIdentifier);
 
     nativeComponent.openSession(sessionIdentifier);
 
-    writeBeginSession(sessionIdentifier, startedAt);
+    writeBeginSession(sessionIdentifier, startedAtSeconds);
```

Comment:
is there any risk, switch to seconds is not granular enough to avoid conflicts / reorderings?

Reply:
It's not a switch, it was always seconds, but the `/ 1000` calculation was just done deeper down. (I see you noticed that https://github.com/firebase/firebase-android-sdk/pull/1296/files#r385929479)